editor/components/MapSelectBasemap.tsx

```diff
@@ -0,0 +1,45 @@
+import { Form } from "react-bootstrap";
+import "mapbox-gl/dist/mapbox-gl.css";
+import { Card } from "react-bootstrap";
+import { Dispatch, SetStateAction } from "react";
+
+interface MapSelectBasemapProps {
+  /** The mapbox basemap type to be used in the map */
+  basemapType: "streets" | "aerial";
+  /** Sets the state for the basemap type */
+  setBasemapType: Dispatch<SetStateAction<"streets" | "aerial">>;
+}
+
+/**
+ * Custom map component for selecting the basemap option
+ */
+export default function MapSelectBasemap({
```

Comment:
would you add `Control` to the end of this name to be consistent with the other control components we have?